src/cascadia/LocalTests_TerminalApp/TerminalApp.LocalTests.AppxManifest.prototype.xml

```diff
@@ -28,7 +28,7 @@
   </Properties>
 
   <Dependencies>
-    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.18362.0" MaxVersionTested="10.0.18362.0" />
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.18362.0" />

```

Comment:
Explain?

Reply:
The CI is running 17763 not 18362. Granted, this particular PR doesn't make the CI work, but if I set this to 18362 then it fails in the CI saying "I can't install this package because it's for a newer version of Windows". I guess this is pre-emptive work that I didn't revert.

Reply:
Ah OK.